File: VideoStreamerUI/src/app/entities/requests/sendChatMessageRequest.ts
import { ChatMessage } from '../chatMessage';

export class SendChatMessageRequest {

  public roomId: string;
  public userId: string;
  public chatMessage: ChatMessage;

  constructor(roomId: string, userId: string, chatMessage: ChatMessage) {
    this.roomId = roomId;
    this.userId = userId;
    this.chatMessage = chatMessage;
  }
}

File: StreamProviderWS/StreamProviderWS/Models/WebSocket/RequestsMessages/SendChatMessageRequest.cs
using StreamProviderWS.Models.Common;

namespace StreamProviderWS.Models.WebSocket.RequestsMessages
{
    public class SendChatMessageRequest
    {
        public string RoomId { get; set; }
        public string UserId { get; set; }
        public ChatMessage ChatMessage { get; set; }


    }
}
